### DISCUSSION TITLE
Supporting WindowsAppSDK >= 1.4

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno#13736] [Q&A] [answered]
Does Uno support WinAppSDK version 1.4.230913002? If I switch from 1.2.221109.1 to that version I get some errors like "The type Application does not exist" and "Unable to find type Microsoft.UI.Xaml.Media.Brush".

Do I need to do some migration or is it not yet supported?

**Comment by limeniye** [comment]
Hello, @TopperDEL! 
I guess this https://github.com/unoplatform/uno/discussions/13569 discussion is similar that you looking for.

**Reply by jeromelaban** [reply, marked as answer]
That's correct, 5.0-dev is fixed, 4.9 and 4.10 will soon be as well.

**Reply by TopperDEL** [reply]
Awesome, thanks!
